matomat/schema.sql

```diff
@@ -0,0 +1,50 @@
+-- Product table
+DROP TABLE IF EXISTS Products;
+CREATE TABLE Products
+(
+  id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+  name TEXT NOT NULL,
+  price REAL NOT NULL,
```

Comment:
Wouldn't it be better if we saved the price as INT?
We could save it as cent and than multiply it by 100 if we need euro. I think real is a little to much for saving money.

Reply:
I gues you are right, shouldn't change much but isn't a lot of work either.
You need >python3.6 in order to run the alpha. You can also check out the docker branch in my repo for a dockerized version of the code. Thus, you don't need to setup your own python env.

Reply:
> check out the docker 

Why do we need docker for a matomat? That sounds a little to much for such a simple device.

Reply:
Its not supposed to be run on the terminal. The terminal should only serve as an endpoint. I'm still talking to AG WASA about hosting it.